client/src/AppContext.js

```diff
@@ -0,0 +1,10 @@
+import { createContext } from "react";
+export const AppContext = createContext({
+	// these values are good for autocompletion
+	userName: "",
+	setUserName: () => {},
+	codewarsData: [],
+	setCodewarsData: () => {},
+	githubData: [],
+	setGitHubData: () => {},
+});
```

Comment:
Github is complaining about not having an empty line at the end of the file here

Reply:
Good point, Fixed